README.md

```diff
@@ -39,31 +96,42 @@ You can pass additional arguments to Theia after the image name, for example to
 
     # Linux, macOS, or PowerShell
     docker run -it --init -p 3000:3000 --expose 9229 -p 9229:9229 -v "$(pwd):/home/project:cached" theiaide/theia:next --inspect=0.0.0.0:9229
-    
+
     # Windows (cmd.exe)
     docker run -it --init -p 3000:3000 --expose 9229 -p 9229:9229 -v "%cd%:/home/project:cached" theiaide/theia:next --inspect=0.0.0.0:9229
 
-### Image Variants
+### Electron Apps
 
-#### `theiaide/theia:latest`
+<img
+    src='./assets/electron.png'
+    alt='electron logo'
+    width="50px"
+/>
 
-This image is based on the latest stable released version.
+| Application Name | Description | Documentation |
+|:---|:---|:---|
+| `theia-cpp-electron` | Theia-based C/C++ desktop IDE | [docs](./theia-cpp-electron/README.md) |
+| `theia-electron` | Theia-based JavaScript/TypeScript (Web Technologies) desktop IDE | [docs](./theia-electron/README.md) |
```

Comment:
Probably important to mention that this is an example of Electron packaging as well, using Electron-Builder